### ISSUE TITLE
Slack Notification ON/OFF toggle

### ISSUE THREAD

**Issue by roratonet** [grokability/snipe-it#9060]
**Server (please complete the following information):**
 - Snipe-IT Version - Latest
 - OS: [e.g. Ubuntu, CentOS] - Debian 10
 - Web Server: [e.g. Apache, IIS] - Nginx
 - PHP Version - Latest working

**Is your feature request related to a problem? Please describe.**
Sometimes when I have to checkin/checkout a lot of assets, I don't want the configured Slack channel with a lot of members to be spammed continuously 

**Describe the solution you'd like**
A toggle in the superadmin interface - ideally under SLACK SETTINGS - where the integration can be switched ON/OFF

**Describe alternatives you've considered**
Same toggle but under NOTIFICATIONS

**Comment by jovibroz** [comment]
Good idea. Would also be awesome with integration per company if full company support is enabled.

**Comment by snipe** [comment]
I'd argue that notifications in general should be company-based, not generalized (unless no companies are present), so this is a UI worker as well as just adding a flag.

**Comment by Godmartinz** [comment]
im going to give this another attempt. I have a vision and a plan for this now.
